blazor/data/adaptors.md

```diff
@@ -213,6 +213,745 @@ The sample response object should look like below.
+        public List<SearchFilter>? Search { get; set; }
+
+        [GraphQLName("Sorted")]
+        public List<Sort>? Sorted { get; set; }
+
+        [GraphQLName("Where")]
+        [GraphQLType(typeof(AnyType))]
+        public List<WhereFilter>? Where { get; set; }
+
+        [GraphQLName("Group")]
+        public List<string>? Group { get; set; }
+
+        [GraphQLName("antiForgery")]
+        public string? antiForgery { get; set; }
+
+        [GraphQLName("Table")]
+        public string? Table { get; set; }
+
+        [GraphQLName("IdMapping")]
+        public string? IdMapping { get; set; }
+
+        [GraphQLName("Select")]
+        public List<string>? Select { get; set; }
+
+        [GraphQLName("Expand")]
+        public List<string>? Expand { get; set; }
+
+        [GraphQLName("Distinct")]
+        public List<string>? Distinct { get; set; }
+
+        [GraphQLName("ServerSideGroup")]
+        public bool? ServerSideGroup { get; set; }
+
+        [GraphQLName("LazyLoad")]
+        public bool? LazyLoad { get; set; }
+
+        [GraphQLName("LazyExpandAllGroup")]
+        public bool? LazyExpandAllGroup { get; set; }
+    }
+
+    public class Aggregate
+    {
+        [GraphQLName("Field")]
+        public string Field { get; set; }
+
+        [GraphQLName("Type")]
+        public string Type { get; set; }
+    }
+
+    public class SearchFilter
+    {
+        [GraphQLName("Fields")]
+        public List<string> Fields { get; set; }
+
+        [GraphQLName("Key")]
+        public string Key { get; set; }
+
+        [GraphQLName("Operator")]
+        public string Operator { get; set; }
+
+        [GraphQLName("IgnoreCase")]
+        public bool IgnoreCase { get; set; }
+    }
+
+    public class Sort
+    {
+        [GraphQLName("Name")]
+        public string Name { get; set; }
+
+        [GraphQLName("Direction")]
+        public string Direction { get; set; }
+
+        [GraphQLName("Comparer")]
+        [GraphQLType(typeof(AnyType))]
+        public object Comparer { get; set; }
+    }
+
+    public class WhereFilter
+    {
+        [GraphQLName("Field")]
+        public string? Field { get; set; }
+
+        [GraphQLName("IgnoreCase")]
+        public bool? IgnoreCase { get; set; }
+
+        [GraphQLName("IgnoreAccent")]
+        public bool? IgnoreAccent { get; set; }
+
+        [GraphQLName("IsComplex")]
+        public bool? IsComplex { get; set; }
+
+        [GraphQLName("Operator")]
+        public string? Operator { get; set; }
+
+        [GraphQLName("Condition")]
+        public string? Condition { get; set; }
+
+        [GraphQLName("value")]
+        [GraphQLType(typeof(AnyType))]
+        public object? value { get; set; }
+
+        [GraphQLName("predicates")]
+        public List<WhereFilter>? predicates { get; set; }
+    }
+
+```
+
+### Performing CRUD action with GraphQLAdaptor
+
+You can perform the CRUD operations by setting the mutation queries in the `Mutation` property of `GraphQLAdaptorOptions`.
```

Comment:
need to explain how we can define the values, and need to mention the methods and arguments that we need to pass and it uses in tabular format